db/schema.rb

```diff
@@ -165,14 +162,15 @@
     t.datetime "updated_at", null: false
     t.decimal "primary_reward", precision: 30, default: "0"
     t.decimal "secondary_reward", precision: 30, default: "0"
-    t.decimal "difficulty", precision: 80, default: "0"
     t.decimal "nonce", precision: 30, default: "0"
     t.decimal "start_number", precision: 30, default: "0"
     t.decimal "length", precision: 30, default: "0"
+    t.integer "uncles_count"
```

Comment:
`uncles_count` haven't removed ?

Reply:
yes, although ckb deleted it from header, Explorer still need it. so I change to calculate it manually.

Reply:
Ok. but `uncles_count` is an u32, `integer` is an i32

Reply:
A normal block should not have too many uncles